### DISCUSSION TITLE
Let users create notes for saved locations

### DISCUSSION THREAD

**Discussion by GaelicGabe** [organicmaps/organicmaps#5991] [Ideas]
One of my frequent problems with OM is that I end up saving a location for some reason and much much later I come back to it and I forgotten what the significance of the location was.
Perhaps it's a restaurant with a very specifically good dish, perhaps it's the code to the bathroom of a hotel, perhaps it's the place I had a romantic outing, perhaps there is a great rock formation here.

I suggest we implement a feature where users are able to save local notes about locations.

What are your guyses thoughts?

**Comment by hb0nd** [comment]
this is already possible :)

https://github.com/organicmaps/organicmaps/assets/26939824/8a64322b-a473-467c-b42d-07a2942498c6

**Reply by GaelicGabe** [reply]
Omg awesome! Thanks for letting me know!
